fdbserver/workloads/Mako.actor.cpp

```diff
@@ -0,0 +1,534 @@
+#include "fdbclient/NativeAPI.actor.h"
+#include "fdbserver/TesterInterface.actor.h"
+#include "workloads.actor.h"
+#include "fdbserver/workloads/BulkSetup.actor.h"
+#include "fdbclient/ReadYourWrites.h"
+#include "flow/actorcompiler.h"
+
+
+enum {OP_GETREADVERSION, OP_GET, OP_GETRANGE, OP_SGET, OP_SGETRANGE, OP_UPDATE, OP_INSERT, OP_INSERTRANGE, OP_CLEAR, OP_SETCLEAR, OP_CLEARRANGE, OP_SETCLEARRANGE, OP_COMMIT, MAX_OP};
+enum {OP_COUNT, OP_RANGE};
+static constexpr int MAXKEYVALUESIZE = 1000;
+static constexpr int KEYPREFIXLEN = 4;
+static constexpr int GETRANGELIMIT = 10000;
+static const char* opNames[MAX_OP] = {"GRV", "GET", "GETRANGE", "SGET", "SGETRANGE", "UPDATE", "INSERT", "INSERTRANGE", "CLEAR", "SETCLEAR", "CLEARRANGE", "SETCLEARRANGE", "COMMIT"};
+static const std::string KEYPREFIX = "mako";
```

Comment:
To add to this, I think using static globals in a cpp-file is bad style (if you must, this is what anonymous namespaces are for). Also, instead of global c-style array, please consider using std::array (you probably can even make it constexpr)

Reply:
Done.